### ISSUE TITLE
Update headers of all files with copyright info

### ISSUE THREAD

**Issue by bocajnotnef** [mercere99/Empirical#21]
something like:

```
// This file is part of Empirical, https://github.com/mercere99/Empirical/, and is 
// Copyright (C) Michigan State University, 2015. It is licensed 
// under the MIT Software license; see doc/LICENSE
```

I think I wrote a script to do this on a per-folder basis...gotta dig it up.

**Comment by bocajnotnef** [comment]
Aha!

```
FILES=/path/to/*.h
for f in $FILES; do  printf '// This file is part of Empirical, https://github.com/mercere99/Empirical/, and is  
// Copyright (C) Michigan State University, 2015. It is licensed                
// under the MIT Software license; see doc/LICENSE\n\n' | cat - $f > temp && mv -f temp $f; done
```

**Comment by bocajnotnef** [comment]
Will be solved by #23

**Comment by bocajnotnef** [comment]
Closing #21, fixed by #23
